src/cbc_binary_toolkit/ubs.py

```diff
@@ -137,12 +137,12 @@ def _download_binary_metadata(cbth, found_binary):
         return {}
 
 
-def _validate_download(cbth, download, expiration_seconds):
+def _validate_download(cbc_api, download, expiration_seconds):
     """
     Verifies the presence of Downloads.FoundItem. Retries downloading if there are errors during download.
 
     Args:
-        cbth (CbThreatHunterAPI): Carbon BlackThreatHunter object.
+        cbc_api (cbc_sdk.CBCloudAPI): Carbon Black Cloud API object.
         download (ThreatHunter.Downloads): May contain found, not_found, and error attributes.
```

Comment:
This should probably be "download (enterprise_edr.Downloads): "

Reply:
Thanks! I thought I caught most but still have a little more to do